External/Plugins/HaXeContext/Context.cs

```diff
@@ -388,6 +388,8 @@ public override void BuildClassPath()
             // add haxe libraries
             if (proj != null)
             {
+                proj.UpdateVars(false);
```

Comment:
This is what happens currently (unless there's a bug somewhere):
When you press the Refresh button in the project panel it sends a `ProjectManager.UserRefreshTree` event which hints HaxeContext to clear the haxelib version cache. Then `BuildClasspath` is called again and it should pick the correct haxelib versions.